python/paddle/fluid/framework.py

```diff
@@ -4148,6 +4164,91 @@ def __init__(self):
         # compiled program, i.e. Graph
         self._graph = None
 
+    def _find_var_ctor_kwargs(self, new_desc):
```

Comment:
what's the meaning of `ctor`? From the code, you are finding 'class' and 'kwargs', can we rename the method name as `_find_var_class_kwargs`?

Reply:
Done.